sdk/iothub/azure-resourcemanager-iothub/src/test/java/com/azure/resourcemanager/iothub/generated/IotHubManagerTests.java

```diff
@@ -0,0 +1,176 @@
+                                    .withName("$fallback")
+                                    .withSource(RoutingSource.DEVICE_MESSAGES)
+                                    .withCondition("true")
+                                    .withIsEnabled(true)
+                                    .withEndpointNames(Arrays.asList("events"))))
+                        .withStorageEndpoints(mapOf("$default",
+                            new StorageEndpointProperties()
+                                .withSasTtlAsIso8601(Duration.ofHours(1L))
+                                .withConnectionString(StringUtil.EMPTY_STRING)
+                                .withContainerName(StringUtil.EMPTY_STRING)))
+                        .withMessagingEndpoints(mapOf("fileNotifications",
+                            new MessagingEndpointProperties()
+                                .withLockDurationAsIso8601(Duration.ofMinutes(1L))
+                                .withTtlAsIso8601(Duration.ofHours(1L))
+                                .withMaxDeliveryCount(10)))
+                        .withEnableFileUploadNotifications(false)
+                        .withCloudToDevice(new CloudToDeviceProperties()
+                            .withMaxDeliveryCount(10)
+                            .withDefaultTtlAsIso8601(Duration.ofHours(1L))
+                            .withFeedback(new FeedbackProperties()
+                                .withLockDurationAsIso8601(Duration.ofMinutes(1L))
+                                .withTtlAsIso8601(Duration.ofHours(1L))
+                                .withMaxDeliveryCount(10)))
+                        .withFeatures(Capabilities.NONE)
+                        .withDisableLocalAuth(false)
+                        .withAllowedFqdnList(Arrays.asList())
+                        .withEnableDataResidency(false)
+                )
+                .create();
+            // embedmeEnd
+            iotHubDescription.refresh();
+
+            Assertions.assertEquals(iotHubDescription.name(), descriptionName);
+            Assertions.assertEquals(iotHubDescription.name(), iotHubManager.iotHubResources().getById(iotHubDescription.id()).name());
+            Assertions.assertTrue(iotHubManager.iotHubResources().list().stream().count() > 0);
+        } finally {
+            if (iotHubDescription != null) {
+                iotHubManager.iotHubResources().deleteById(iotHubDescription.id());
+            }
+        }
+    }
+
+    private static String randomPadding() {
+        return String.format("%05d", Math.abs(RANDOM.nextInt() % 100000));
+    }
+
+    @SuppressWarnings("unchecked")
+    private static <T> Map<String, T> mapOf(Object... inputs) {
+        Map<String, T> map = new HashMap<>();
+        for (int i = 0; i < inputs.length; i += 2) {
+            String key = (String) inputs[i];
+            T value = (T) inputs[i + 1];
+            map.put(key, value);
+        }
+        return map;
+    }
```

Comment:
Don't copy this. Use a Map/HashMap and do normal map.put(). Just write sane Java code.

This thing is only for sample.

Reply:
Fixed in the new version.